site/content/docs/user/quick-start.md

```diff
@@ -218,25 +218,23 @@ context name `kind` and delete that cluster.
 
 ## Loading an Image Into Your Cluster
 
-You can load one or more images into your kind cluster:
+Docker images can be loaded into your cluster nodes with:
 
-```bash
-kind load docker-image my-app:latest
-```
-
-```bash
-kind load docker-image my-app:latest my-db:latest my-cache:latest
-```
-
-Note: If using a named cluster you will need to specify the name of the cluster:
+`kind load docker-image my-custom-image-0 my-custom-image-1`
 
-```bash
-kind load docker-image my-app:latest --name test-cluster
-```
+> **Note**: If using a named cluster you will need to specify the name of the
+> cluster you wish to load the images into:
+> `kind load docker-image my-custom-image-0 my-custom-image-1 --name kind-2`
 
 Additionally, image archives can be loaded with:
 `kind load image-archive /my-image-archive.tar`
 
+For additional usage details and available flags, you can run:
```

Comment:
Maybe instead we should document somewhere that `kind help` exists.

That recursively documents `--help` and using it for subcommands. I don't think we want to add an entry like this to the docs for every command.